src/types/cuid.ts

```diff
@@ -16,20 +16,20 @@ class SchemaTypeCUID extends SchemaType<string> {
    */
   cast(value?: string): string {
     if (value == null && this.options.required) {
-      return cuid();
+      return nanoid();
     }
 
     return value;
   }
 
   /**
-   * Validates data. A valid CUID must be started with `c` and 25 in length.
+   * Validates data. A valid CUID must be 21 in length.
    *
    * @param {*} value
    * @return {String|Error}
    */
   validate(value?: string): string {
-    if (value && (value[0] !== 'c' || value.length !== 25)) {
+    if (value && (value.length !== 21)) {
       throw new ValidationError(`\`${value}\` is not a valid CUID`);
     }
```

Comment:
Since the schema of the id has changed, pre-PR's `db.json` will become invalid. It is a breaking change that we might want to consider about.

Reply:
That makes sense. I will change its format to be compatible with old cuid.